### PR TITLE
Final 1.15 EOL Announcement

### PR DESCRIPTION
**Pull request by dhawton**
Please provide a description for what this PR is for.

In preparation for next Tuesday's EOL date.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/support/announcing-1.15-eol-final/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Support for Istio 1.15 has ended
+subtitle: Support Announcement
+description: Istio 1.15 end of life announcement.
+publishdate: 2023-03-28
```
[the remark this answers is not in the document]
Comment (dhawton) [reply content/en/news/support/announcing-1.15-eol-final/index.md]: Done

File: data/compatibility/supportStatus.yml [hunk 2]
```diff
@@ -20,9 +20,9 @@
   k8sVersions: ["1.22", "1.23", "1.24", "1.25"]
   testedK8sVersions: ["1.16", "1.17", "1.18", "1.19", "1.20", "1.21"]
 - version: "1.15"
-  supported: "Yes"
+  supported: "No"
   releaseDate: "August 31, 2022"
-  eolDate: "~March 2023 (Expected)"
+  eolDate: "March 28, 2023"
```
[the remark this answers is not in the document]
Comment (dhawton) [reply data/compatibility/supportStatus.yml]: Commit to adjust to April matches EOL dates used elsewhere. Wondering why we use full month in release date and shortened months in EOL dates.

[the remark this answers is not in the document]
Comment (ericvn) [reply data/compatibility/supportStatus.yml]: @dhawton Good question, and I don't know the answer. I assume it was something related to trying to keep the year on the same line as the date, but then why not for both? Maybe before some rewrite the column widths were different? Maybe @craigbox has some thoughts. I would be inclined to make them all numerical like 2023-04-04 and see if that makes it a single row of text in each row, and would be consist across columns, but see if Craig mentions anything first.

[the remark this answers is not in the document]
Comment (craigbox) [reply data/compatibility/supportStatus.yml]: no reason I can think of.  someone just did it that way and everyone followed, I guess.  Feel free to change in any way you like.

[the remark this answers is not in the document]
Comment (dhawton) [reply data/compatibility/supportStatus.yml]: Opened https://github.com/istio/istio.io/pull/12954 to address this and open a discussion about.

**Comment by ericvn** [comment]
This was included in the CVE docs.

**Comment by dhawton** [comment]
Now only updating the supported releases table which wasn't part of the CVE announcement PR.

**Review by ericvn — APPROVED** [review]

**Comment by ericvn** [comment]
Thanks Daniel.

**Pull request merged**